common/src/main/java/org/apache/gravitino/dto/rel/PartitionUtils.java

```diff
@@ -38,7 +38,12 @@ public class PartitionUtils {
   public static void validateFieldExistence(ColumnDTO[] columns, String[] fieldName)
       throws IllegalArgumentException {
     Preconditions.checkArgument(ArrayUtils.isNotEmpty(columns), "columns cannot be null or empty");
-
+    Preconditions.checkArgument(
+        fieldName != null
+            && fieldName.length > 0
+            && fieldName[0] != null
+            && !fieldName[0].trim().isEmpty(),
+        "fieldName[0] cannot be null, empty or blank");
```

Comment:
Maybe the following is sufficient.
```
Preconditions.checkArgument(ArrayUtils.isNotEmpty(fieldName), "fieldName cannot be null or empty");
```

Anyway,  the current implementation is more strict and I find it acceptable .